### PR TITLE
feat: integrate VIDIS

### PR DESCRIPTION
**Pull request by hugotiburtino**
There is a way of using the a web component through a CDN, could we try it out? 
I suppose it is important for brand UI button.
```
VIDIS-Login Example
<script src="https://repo.vidis.schule/repository/vidiscdn/latest/vidisLogin.umd.js"></script>
<vidis-login loginurl="https://www.domain.de/path-toauth/" size="L" cookie="true"></vidis-login>
```

### CODE REVIEW

File: apps/web/src/components/pages/auth/logout.tsx [hunk 1]
```diff
@@ -32,23 +32,30 @@ export function Logout({ oauth }: { oauth?: boolean }) {
...
+      ? cookieSession.authentication_methods[0].provider
+      : undefined
 
-    const redirection = isSSO
-      ? 'https://aai.demo.meinbildungsraum.de/realms/nbp-aai/protocol/openid-connect/logout'
-      : filterUnwantedRedirection({
-          desiredPath: sessionStorage.getItem('previousPathname'),
-          unwantedPaths: [settingsUrl, loginUrl, registrationUrl],
-        })
+    const redirection =
```
Comment (shn-srl) [inline apps/web/src/components/pages/auth/logout.tsx:39]: Nitpick: Can we make this a key value map or something cleaner than the nested ternary's?

Comment (hugotiburtino) [reply apps/web/src/components/pages/auth/logout.tsx:39]: I've just kept the style convention of the repo, but sure you can refactor it. I think a function would be the most readable refactoring.

**Comment by hugotiburtino** [comment]
If the integration of the vidis webcomponent isn't that easy, could we merge to start testing it and do the changes afterwards?

**Comment by hugotiburtino** [comment]
I'll merge it and do the improvements in other PR

**Pull request merged**